codecov.yml

```diff
@@ -1,6 +1,12 @@
+# github_checks:
+#     annotations: false  # remove annoying inline code comments
 coverage:
   status:
     project: off
     patch:
       default:
+        # informational: true  # turn this one on to get cov tests to always pass
```

Comment:
I agree with @jvegasbsc and @zklaus that we should keep the failed test, it already led to a bit improvement in coverage in #1233.

Reply:
OK I agree too, after convincing myself, with help from you guys :laughing: That's whay I commented out the lines; BTW I think a few of these commented out lines and in-line comments with what they do are useful in the future if we want to re-activate/de-activate options

Reply:
Great! Personally, I'm not a big fan of leaving commented-out stuff everywhere, it looks distracting to me, like the author was working on something and then got the code merged without cleaning up afterwards.